travis/run-large-python-tests.sh

```diff
@@ -15,6 +15,10 @@ else
   cat $SAGEMAKER_OUT;
 fi
 
+# Set matplotlib to not use the Xwindows backend which causes an error.
```

Comment:
Why set this config here instead of programmatically in test_xgboost_autolog as was done previously? The only issue with setting the config here is that if someone runs this script locally while developing a PR, we'll alter their matplotlib settings

Reply:
@smurching I agree. I reverted the changes.